libs/sdk-core/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 - `/ephemeral_tokens` endpoint can now be called using regional domain
 - Support `summarize` operation on resources
-- Remove `onlyAnswers` property from `WidgetFeatures` interface, only answers is now managed by `hideSources` option. 
+- Remove `onlyAnswers` property from `WidgetFeatures` interface, only answers is now managed by `hideSources` option.
+- Support the `citation` option on `/chat` endpoint
```

Comment:
it would probably worth a new SDK version no? specially knowing the last one was from last month.